keyboards/varanidae/varanidae.c

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2023 QMK
+ * Copyright 2023 Silc Renew / Tokage IT Lab.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) {
+        return false;
+    }
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_MS_WH_DOWN);
+        } else {
+            tap_code(KC_MS_WH_UP);
+        }
+    }
+    return true;
+}
+#endif
```

Comment:
Isn't strictly needed now, as there is default behavior, unless you want to explicitly set it as a mousewheel by default.  Otherwise, it's volume control if extra keys are enabled, or scroll keys if mousekeys are enable but extrakeys are not.

Reply:
Since the concept of this keyboard was to be developed for working with Blender, it is intended to function as a wheel scroll in most cases. Since we hold the pen tablet in our right hand, we needed the scroll wheel somewhere to change blender inspector value.

Reply:
Adopted .encoder_map`.